### ISSUE TITLE
Feature Request: Embed minicards 

### ISSUE THREAD

**Issue by yelloff** [wekan/wekan#3456]
We use discourse and wekan as our main organizing tools. So I would like to embed certain minicards in discourse posts. Like a onebox preview of the particular minicard.

**Comment by xet7** [comment]
@yelloff 

Do you mean embedding one card of a public board? That could be easier to implement with some special URL.

Or would you need embedding one card of a private board? That would need permissions etc setup.

Where is onebox documentation?

**Comment by xet7** [comment]
Is it this? https://github.com/discourse/onebox

Should that embed be like html page? Or XML ?

**Comment by yelloff** [comment]
> Is it this? https://github.com/discourse/onebox
> 
> Should that embed be like html page? Or XML ?

Yes, thats the "onebox". Dicourse use it to show html preview of certain websites. For our particular needs it doesn't matter if the embedding card is of a public or of a private board. And in my ideal world a card should be embedded as html.

**Comment by xet7** [comment]
@yelloff 

Do you have a possibility to pay something for that feature?
https://wekan.team/commercial-support/

**Comment by yelloff** [comment]
> @yelloff
> 
> Do you have a possibility to pay something for that feature?
> https://wekan.team/commercial-support/

Yes.

**Comment by xet7** [comment]
@yelloff

Ok. After you have paid something, I can try to implement this.

**Comment by xet7** [comment]
This feature request [has been added to Roadmap](https://boards.wekan.team/b/D2SzJKZDS4Z48yeQH/wekan-open-source-kanban-board-with-mit-license/Xo9no7jcjxt2wwLWj), payment process in in progress.

**Comment by yelloff** [comment]
How is it going with this one?

**Comment by xet7** [comment]
@yelloff 

To have possibility to develop this, it needs first Multiverse speedups that are targeted to end of this month. So maybe in November? Just guessing.

**Comment by xet7** [comment]
@yelloff 

In theory, it could be possible in Meteor WeKan, if it would be done similar like REST API returning JSON, but instead retuning small amount of HTML/CSS. But because such code does not yet exist for Meteor WeKan, most probable for it is to be in Multiverse WeKan.
